app/forms/developer_search_form.rb

```diff
@@ -4,8 +4,16 @@ class DeveloperSearchForm
   include Virtus.model
   include ActiveModel::Model
 
-  attribute :programming_language_id, Integer
-  attribute :language_id, Integer
+  FORM_FIELDS = %w[programming_language_id language_id].freeze
```

Comment:
Just in case, I think the previous implementation is ok.
Although you should write like this.
```ruby
attribute :programming_language_id, Integer
attribute :language_id, Integer
FORM_FIELDS = %w[programming_language_id language_id].freeze
```

Reply:
Fixed!